clang/lib/Sema/SemaSYCL.cpp

```diff
@@ -366,15 +385,6 @@ static void collectSYCLAttributes(Sema &S, FunctionDecl *FD,
       FD->dropAttr<SYCLIntelUseStallEnableClustersAttr>();
     }
   }
-
-  // Attributes that should not be propagated from device functions to a kernel.
-  if (DirectlyCalled) {
```

Comment:
These should be copied in SYCL 1.2.1 as well if directly applied.

Reply:
Done